eixampledb-core/pom.xml

```diff
@@ -20,5 +20,11 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Spring already has a Json library called Jackson. We are going to change this in the future.